### PR TITLE
boveri Analyte node updates

### PR DESCRIPTION
**Pull request by ganjigajanan**

### CODE REVIEW

File: gdcdictionary/schemas/analyte.yaml [hunk 1]
```diff
@@ -111,11 +114,13 @@ properties:
     $ref:
       - "_terms.yaml#/analyte_volume/common"
     type: number
+    minimum: 0
```
Comment (wteouchicago) [inline gdcdictionary/schemas/analyte.yaml:117]: didn't spot this in the ticket (DICT-100), but it makes sense.

File: gdcdictionary/schemas/analyte.yaml [hunk 2]
```diff
@@ -168,11 +173,13 @@ properties:
     $ref:
       - "_terms.yaml#/ribosomal_rna_28s_16s_ratio/common"
     type: number
+    minimum: 0
 
   rna_integrity_number:
```
Comment (wteouchicago) [inline gdcdictionary/schemas/analyte.yaml:178]: ticket has a min (DICT-100)

File: gdcdictionary/schemas/analyte.yaml (hunk 1 above)
Comment (ganjigajanan) [reply gdcdictionary/schemas/analyte.yaml:117]: confirmed with Bill as well

**Review by wteouchicago — CHANGES_REQUESTED** [review]
well_number changes are in the ticket but not here.

File: gdcdictionary/schemas/analyte.yaml (hunk 2 above)
Comment (wteouchicago) [reply gdcdictionary/schemas/analyte.yaml:178]: oops - that was for rna_integrity_number

Comment (ganjigajanan) [reply gdcdictionary/schemas/analyte.yaml:178]: Confirmed with Bill the rna_integrity_number minimum value mentioned in ticket needs review. hence did not add it to node. can be added in future if needed after confirming

**Review by wwysoc2 — APPROVED** [review]

**Review by wwysoc2 — APPROVED** [review]

**Comment by wwysoc2** [comment]
We decided not to do the well number changes.  @wteouchicago could you consider the changes done?  I can't figure out how to do that.

**Review by wteouchicago — APPROVED** [review]
my concerns have been addressed.

**Pull request merged**